DataImport.Models/Migrations/SqlServer/20240419161441_Agent_FilesAction.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
```

Comment:
@mjimdev General rule (I think) is that we don't have schema changes during a non-major release. Will need verification from @jasonh-edfi and team that this schema change is possible for this release